docs/src/main/paradox/pravega.md

```diff
@@ -117,6 +129,43 @@ Java
 It produces a stream of @apidoc[PravegaEvent], a thin wrapper which includes some Pravega metadata along with the
 message.
 
+### Key Value Pair table
```

Comment:
Can you add appropriate Pravega docs references to the Key Value Pair table concepts here or at the beginning of the page?